gulpfile.js

```diff
@@ -34,7 +34,8 @@ gulp.task("templates", function() {
         }))
         .pipe(defineModule("plain"))
         .pipe(declare({
-            namespace: "Handlebars.templates"
+            namespace: "Handlebars.templates",
+            root: "window"
```

Comment:
This makes it so that all the Handlers.templates definitions use "window" instead of "this" (which resolves to undefined in Webpack land).